Assets/MixedRealityToolkit/SpatialUnderstanding/README.md

```diff
@@ -9,7 +9,7 @@ or in your Visual Studio Package.appxmanifest capabilities.
 
 ### Microsoft HoloLens Documentation
 - [Spatial Mapping In Unity](https://developer.microsoft.com/en-us/windows/mixed-reality/spatial_mapping_in_unity)
-- [Spatial Understanding In Unity](https://developer.microsoft.com/en-us/windows/mixed-reality/spatial_mapping_in_unity#MixedRealityToolkit.spatialunderstanding)
+- [Spatial Understanding In Unity](https://developer.microsoft.com/en-us/windows/mixed-reality/spatial_mapping_in_unity#holotoolkit.spatialunderstanding)
```

Comment:
I think this has actually been updated to https://developer.microsoft.com/en-us/windows/mixed-reality/spatial_mapping_in_unity#higher-level_mesh_analysis:_spatialunderstanding

Reply:
Updated to new link